app/src/main/res/drawable/samlpl.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Name of this file isn't suggesting anything. Give it a more descriptive name

Reply:
It was for testing. Deleted it